test/integration/pdf_generation_test.exs

```diff
@@ -402,8 +415,8 @@ defmodule ChromicPDF.PDFGenerationTest do
       ]
 
       assert capture_log(fn ->
-               assert_raise ChromicPDF.Browser.ExecutionError,
-                            ~r/Timeout in Channel.run_protocol/,
+               assert_raise ChromicPDF.ChromeError,
+                            ~r/Chrome returned an error/,
```

Comment:
I'm not sure about this one. Is this also fine?

Reply:
Yes, changing the error is fine here, the test should assert on the log statement.